source/jobs/_contact_us.erb

```diff
@@ -5,7 +5,7 @@
     </h2>
 
     <p class="text-tile__paragraph--with-spacing">
-      Steve handles recruitment for Unboxed and would love to hear from you.
+    Steve is our people person. If <%= (roles > 1)? "any of these roles" : "this role" %> sounds of interest to you, he would love to hear from you.
```

Comment:
Sorry to nitpick but in the case where there's more than one role I'm not sure if it's grammatical..

maybe '...*one* of these roles sounds of interest`?